src/js/features/classification/taxonomy-controls.js

```diff
@@ -158,7 +158,7 @@ const TaxonomyControls = ( { onChange, query } ) => {
 								// Append newTerm to taxoInfo.terms.
 								const terms = {
 									...taxoInfo.terms,
-									entitites: [
+									entities: [
```

Comment:
It seems like this typo may have caused problems but would be curious to know if that was the case

Reply:
`typos` is a very useful tool. Not every day but the benefits are great on those days.

Reply:
BTW `tsc` from Typescript does perform [static analysis on JavaScript](https://github.com/szepeviktor/byte-level-care/issues/245) too.

Reply:
@dkotter, I just checked this. Due to a typo, the `terms` object was updated with a new key, but since we are not using the `entities` data, it hasn't caused any issues yet.